src/lib/optimizer/strategy/between_composition_rule.cpp

```diff
@@ -36,6 +36,11 @@ PredicateCondition get_between_predicate_condition(bool left_inclusive, bool rig
 
 namespace opossum {
 
+std::string BetweenCompositionRule::name() const {
+  static const auto name = std::string{"BetweenCompositionRule"};
```

Comment:
Would it make sense to use proper names with spaces here?

Reply:
> Would it make sense to use proper names with spaces here?

I do not see any benefits.

> How is this done for our operators? Do we want to have it analogously?

Yes. `name()` outputs the operator's class name in PascalCase.

Reply:
> I do not see any benefits.

Me neither.

Reply:
Well, it's the way the english language works. :flushed: